tests/testthat/test-dataframe.R

```diff
@@ -424,3 +424,10 @@ test_that("to_Struct, unnest, to_frame, as_data_frame", {
   expect_identical(df$as_data_frame(), df_e)
 })
 
+
```

Comment:
you should add a similar test in test-lazy.R to formally test lazy tail

Reply:
done